zuul.d/projects.yaml

```diff
@@ -94,11 +99,30 @@
     templates:
       - system-required
 
+- project:
+    name: github.com/ansible-community/molecule-gce
+    templates:
+      - system-required
+
+- project:
+    name: github.com/ansible-community/molecule-hetznercloud
+    templates:
+      - system-required
+
 - project:
     name: github.com/ansible-community/molecule-libvirt
     templates:
       - system-required
-      - publish-to-pypi
```

Comment:
you want to stop publishing to pypi here?

Reply:
No, but considering that publishing worked for molecule itself which does not have it, I assumed is not needed, being included in system ones. All repos should have it or not, this one was missing it.